pysrc/bytewax/inputs.py

```diff
@@ -30,6 +30,7 @@ def tumbling_epoch(
     epoch_length: Any,
     time_getter: Callable[[Any], Any] = lambda _: datetime.datetime.now(),
     epoch_0_start_time: Any = None,
+    epoch_start: Any = 0,
```

Comment:
We don't support any other type of epoch than `int` so you could make the type hint `: int`.